manager/controllers/advisory_detail.go

```diff
@@ -21,16 +22,16 @@ type AdvisoryDetailItem struct {
 }
 
 type AdvisoryDetailAttributes struct {
-	Description  string    `json:"description"`
-	ModifiedDate time.Time `json:"modified_date"`
-	PublicDate   time.Time `json:"public_date"`
-	Topic        string    `json:"topic"`
-	Synopsis     string    `json:"synopsis"`
-	Solution     string    `json:"solution"`
-	Severity     *int      `json:"severity"`
-	Fixes        *string   `json:"fixes"`
-	Cves         []string  `json:"cves"`
-	References   []string  `json:"references"`
+	Description  string          `json:"description"`
+	ModifiedDate time.Time       `json:"modified_date"`
+	PublicDate   time.Time       `json:"public_date"`
+	Topic        string          `json:"topic"`
+	Synopsis     string          `json:"synopsis"`
+	Solution     string          `json:"solution"`
+	Severity     *int            `json:"severity"`
+	Fixes        *string         `json:"fixes"`
+	Cves         *postgres.Jsonb `json:"cves"`
```

Comment:
Ise this serialized correctly as nested object ? Or is it serialized as a string containing json ?

Reply:
I've tried, it returns correct nested object:
~~~json
            "fixes": null,
            "cves": [
                "CVE-1",
                "CVE-2"
            ],
            "references": []
~~~